lang/funcs/collection_test.go

```diff
@@ -1469,6 +1469,22 @@ func TestLookup(t *testing.T) {
 			cty.StringVal("baz"),
 		}),
 	})
+	mapOfMaps := cty.MapVal(map[string]cty.Value{
+		"foo": cty.MapVal(map[string]cty.Value{
+			"a": cty.StringVal("bar"),
+		}),
+		"baz": cty.MapVal(map[string]cty.Value{
+			"b": cty.StringVal("bat"),
+		}),
+	})
+	mapOfObjects := cty.ObjectVal(map[string]cty.Value{
```

Comment:
This seems to be an "object of maps", rather than the other way around.

Reply:
Ah, I noticed that too, but forgot to fix it. I bet I wrote that in the first place 🙃